src/contentType.ts

```diff
@@ -0,0 +1,35 @@
+import { fileTypeFromBuffer } from 'file-type'
+import mime from 'mime-types'
+
+interface testInput {
+  bytes: Uint8Array
+  path: string
+}
+
+type testOutput = Promise<string | undefined>
+
+export const DEFAULT_MIME_TYPE = 'text/html'
+
+const tests: Array<(input: testInput) => testOutput> = [
```

Comment:
Might be useful to abstract the content-type tests to a type/interface and use that.